lib/manager/gradle/build-gradle.ts

```diff
@@ -387,3 +387,36 @@ export function updateGradleVersion(
   }
   return buildGradleContent;
 }
+
+const additionalKeywords = ['mavenBom'].join('|');
+
+const additionalKeywordsRegex = regEx(
+  `[\\s\\n{](${additionalKeywords})\\s*['"](?<depName>[a-zA-Z][-_a-zA-Z0-9.]*?:[a-zA-Z][-_a-zA-Z0-9.]*?):(?<currentValue>[a-zA-Z0-9][-.a-zA-Z0-9]*?)['"]`
+);
+
+export function extractAdditionalDeps(
+  content: string,
+  deps: BuildDependency[]
+): void {
+  const registryUrls = deps[0]?.registryUrls;
+  for (const line of content.split(/\s*\n+/)) {
+    const match = additionalKeywordsRegex.exec(line);
+    if (match) {
+      const { depName, currentValue } = match.groups;
+      const [name, depGroup] = depName.split(':');
+      if (
+        !deps.some(
+          (dep) => dep.depName === depName && dep.currentValue === currentValue
+        )
+      ) {
+        deps.push({
+          name,
+          depGroup,
+          depName,
+          currentValue,
+          ...(registryUrls && { registryUrls }),
```

Comment:
Can you make this easier to understand? Maybe expand it into 2-3 lines if necessary so that I'm not the only one confused by it in future :)